 <launch>
   <node pkg="jackal_affordance" type="affordance_detect" name="hsr_affordance_detect" output="screen"> <!--launch-prefix="xterm -geometry 170x30+10+20 -e gdb -x=$(find jackal_affordance)/debug/gdb_jackal_affordance -args" -->
     <rosparam>
       info_topic: /hsrb/head_rgbd_sensor/rgb/camera_info
       rgb_topic: /hsrb/head_rgbd_sensor/rgb/image_rect_color
       point_cloud_topic: /hsrb/head_rgbd_sensor/depth_registered/rectified_points
       fixed_frame: /map
       min_primitive_size: 100
       max_plane_area: 5
       min_seg_size: 200
       seed_resolution: 0.3
       voxel_resolution: 0.03
     </rosparam>
   </node>
 </launch>
